Fix #20649 (#20724)
kubernetes/website: content/en/docs/concepts/scheduling-eviction/eviction-policy.md

---
title: Eviction Policy
content_template: templates/concept
weight: 50
---

{{% capture overview %}}

> This page is an overview of the Eviction Policy used by `kubelet`.

Reviewer: nit (OK to merge without fixing): `kubelet` is usually written without backticks.

On the wording, I suggest:
```suggestion
This page is an overview of Kubernetes' policy for eviction.
```

{{% /capture %}}

{{% capture body %}}

> ## Eviction Policy

Reviewer 2: Note that eviction can be triggered by other components: the scheduler during preemption, also I think node lifecycle controller evicts pods when NoExecute taints are violated.

Reviewer: _for now_ I'm happy with a simple copy-and-paste. My thinking was to then log another issue to capture the need for more subtle tidying.
